src/plugins/visualizations/public/wizard/dialog.scss

```
@@ -1,84 +1,57 @@
-  background-repeat: no-repeat;
-  background-position: calc(100% + 1px) calc(100% + 1px);
-  background-size: 30%;
-}
+$modalWidth: $euiSizeL * 34;
+$modalHeight: $euiSizeL * 29;
```

I noticed that there's a scrollbar appearing because the height of the inner cards grew with the new texts. So to avoid this scrollbar we can increase de modal height:

 ```suggestion
$modalHeight: $euiSizeL * 30;
```